utils/merge.py

```
@@ -1,91 +1,166 @@
+        "--log-level",
+        env_log_level,
+    ]
+
+    logger.info(f"Running: {' '.join(cmd)}")
 
     try:
-        logger.info(f"Processing {book.title}")
-        m4b.run_merge()
-    except Exception as e:
-        logger.error(f"Error occured while merging '{input_data}: {e}'")
+        result = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)